src/javascript/app/App.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import '@elastic/react-search-ui-views/lib/styles/styles.css';
-import JahiaSearchAPIConnector, {Field, FieldType} from '@jahia/search-ui-jahia-connector';
+import JahiaSearchAPIConnector, {Field, FieldType} from '../../../../search-ui-jahia-connector/src';
```

Comment:
You could use yalk to keep the import name the way it is, only package.json import would change. As it stands right now nobody would be able to run the module unless there is the same file structure.

Reply:
Same, this was one of the first commits, I changed it to be good I guess)